test/integration/site_layout_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+
+class SiteLayoutTest < ActionDispatch::IntegrationTest
+  test "layout links" do
+    get root_path
+    assert_template 'static_pages/home'
+    assert_select "a[href=?]", root_path, count: 2
+    assert_select "a[href=?]", help_path
+    assert_select "a[href=?]", about_path
+    assert_select "a[href=?]", contact_path
+  end
+end
```

Comment:
[Q]
We have one more test about `ApplicationHelperTest` do we need to add this ?